doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -253,12 +276,11 @@ The subsequent subsections describe required steps for enabling Fast Pair extens
 Enabling Fast Pair in Kconfig
 =============================
 
-If you are using the default |NCS| build system configuration with sysbuild and wish to add the Google Fast Pair subsystem to your project, enable the ``SB_CONFIG_BT_FAST_PAIR`` Kconfig option.
-If you do not use sysbuild, you must enable :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option at the main application image level.
+To support Google Fast Pair in your project, you must enable :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option at the main application image level.
 
 .. note::
-   Sysbuild sets the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option in the main application image based on the value of the ``SB_CONFIG_BT_FAST_PAIR`` Kconfig option.
-   Your configuration of the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option at the main application image will be ineffective, as sysbuild overrides it.
+   The Fast Pair Kconfig configuration at the sysbuild level is only used to generate the Fast Pair provisioning data hex file during the build run.
```

Comment:
```suggestion
   The Fast Pair Kconfig configuration at the sysbuild level is only used to generate the Fast Pair provisioning data hex file during the build.
```

Reply:
Consider mentioning here also that the provisioning is automatically merged with the `merged.hex` file (to provide complete information)

Reply:
I cannot possibly copy all the sentences from the previous sections. I already provided this information in the relevant section. You have a ref link here. This section is about setting Fast Pair Kconfigs. Why should we describe the hex upload process here?